src/SmartComponents/Applications/Applications.js

```diff
@@ -23,7 +58,24 @@ Applications.propTypes =  {
         params: PropTypes.shape({
             id: PropTypes.string
         })
-    })
+    }),
+    getSchema: PropTypes.func,
+    loaded: PropTypes.bool,
+    schema: PropTypes.object
 };
 
-export default Applications;
+function mapStateToProps({ applicationsStore }) {
+    return {
+        schema: applicationsStore && applicationsStore.schema,
+        loaded: applicationsStore && applicationsStore.loaded
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+        getSchema: (application) => dispatch(getSchema(application))
+    };
+}
+
+const ApplicationsConnected =  connect(mapStateToProps, mapDispatchToProps)(Applications);
+export default ApplicationsConnected;
```

Comment:
No need to assign to variable `export default connect(mapStateToProps, mapDispatchToProps)(Applications)`
```suggestion
export default connect(mapStateToProps, mapDispatchToProps)(Applications);
```